services/userService.js

```diff
@@ -0,0 +1,64 @@
+const userDao = require('../models/userDao');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const saltRounds = 10;
```

Comment:
```suggestion
const bcrypt = require('bcrypt');
const jwt = require('jsonwebtoken');

const userDao = require('../models/userDao');
```
built-in 모듈 -> 외부 모듈 -> 커스텀 모듈 순으로 띄어쓰기를 활용해 구분하면서 import 해주세요~

전역 변수로 선언된 SaltRounds는 이를 사용하는 함수 내의 scope에 선언하거나, 아니면 단순 값이기에 직접적으로 넣어도 상관없을 것 같습니다.